README-MULTISITE.md

```diff
@@ -79,6 +79,10 @@ system_secret_key: MGecsMrWtKZgngOHZdrd6d3JxGO5CPWgT2lcnpSt
 rgw_pull_proto: http
 rgw_pull_port: 8080
 rgw_pullhost: cluster0-rgw0
+rgw_primary_endpoints:
```

Comment:
Does this need to be a static variables ?
I'm just wondering if we could generate this list dynamically from the rgw deployed. WDYT ?